VMS/Pages/Organization-Profile/ActTookPlace.razor

```diff
@@ -0,0 +1,155 @@
+﻿<div class="container">
+    <div class="row act">
+        <h1>Hoạt động <span>đã diễn ra</span></h1>
+        <div class="row pb-5">
+            <div class="col-6 act-container position-relative">
+                <div class="img-container position-absolute">
+                    <img src="./img/OrganizationProfile/ActTookPlace.png" alt="" class="act-img">
+                </div>
+                <div class="act-content col-10 ms-auto my-1">
+                    <div class="act-ctn col-9 ms-auto ps-4 pe-3 mt-3 mb-2">
+                        <a href ="" class="act-title">Tuyển tình nguyện viên tham gia hỗ trợ công tác phòng, chống dịch Covid-19</a>
```

Comment:
We don't hard code those values like this anymore, please bind data using foreach loop instead.